### PR TITLE
5275 set multiple images as desktop background

### LINKED ISSUE #5275: "Set as" Doesn't appear when selecting >1 image.
"Set as" does not appear on the right click menu when **multiple** images are selected.

When trying to create a slideshow background, on explorer I would: 
Highlight multiple images > right click > Set as > Background

System Information
 - OS Version: Windows 10 19043.1055 21H1 
 - App version: v1.5

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #5275.

**Details of Changes**
Add details of changes here.
- Added the "Set as desktop slideshow" command in the ContextFlyout menu.
- Added bigger constraints on "Set as Desktop Background" and "Set as lockscreen background" so they appear only if one image is selected.
- Readapted _CheckFileExtension_ in _src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs_ to be able to handle multiples extensions in entry, reflecting any multi-selection.
- Did some light refactor on _src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs_.

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots**

SetAsDesktopSlideshow

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@puppetsw does your PR also do this?

**Comment by puppetsw** [comment]
> @puppetsw does your PR also do this?

Only for the toolbar not the context menu.

File: src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs [hunk 1]
```diff
@@ -537,29 +538,25 @@ public bool IsSelectedItemShortcut
...
         }
 
-        public void CheckFileExtension(string itemExtension)
+        public void CheckAllFileExtensions(List<string> itemExtensions)
         {
-            // Set properties to false
-            IsSelectedItemImage = false;
-            IsSelectedItemShortcut = false;
+            // Checks if all the item extensions are image extensions of some kind.
+            IsSelectedItemImage = itemExtensions.TrueForAll(itemExtension => this.IsImageExtension(itemExtension));
```
Comment (puppetsw) [inline src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs:544]: In the `FileExtensionHelpers` there an `IsImageFile` method that can be used.

File: src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs [hunk 2]
```diff
@@ -537,29 +538,25 @@ public bool IsSelectedItemShortcut
...
+            // Checks if there is only one selected item and if it's a shortcut.
+            IsSelectedItemShortcut = (itemExtensions.Count == 1) && (itemExtensions.TrueForAll(itemExtension => this.IsShortcutExtension(itemExtension)));
```
Comment (puppetsw) [inline src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs:546]: Might be worthwhile moving the `IsShortcutExtension` method into the `FileExtensionHelpers` class as well?

**Review by puppetsw — CHANGES_REQUESTED** [review]

File: src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs (hunk 1 above)
Comment (QuaintMako) [reply src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs]: Done!

File: src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs (hunk 2 above)
Comment (QuaintMako) [reply src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs]: Done!

**Review by puppetsw — APPROVED** [review]
LGTM

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**